libnavigation-core/src/main/java/com/mapbox/navigation/core/MapboxNavigation.kt

```diff
@@ -40,6 +40,16 @@ import kotlinx.coroutines.channels.ReceiveChannel
 class MapboxNavigation(
     private val context: Context,
     private val accessToken: String?,
+    private val navigationOptions: NavigationOptions = NavigationOptions.Builder(
+        TWELVE_HOURS,
+        ROUNDING_INCREMENT_FIFTY,
+        distanceFormatter = MapboxDistanceFormatter(
+            context.applicationContext,
+            "ja",
```

Comment:
Yeah as @LukasPaczos mentioned https://github.com/mapbox/mapbox-navigation-android/pull/2419/files#r370624324 these values were hardcoded for testing purposes.